dags/analyzer_helper/discourse/extract_raw_info.py

```diff
@@ -0,0 +1,275 @@
+        # MATCH (topic:DiscourseTopic)
+        # WHERE id(topic) = post.topicId
+        # OPTIONAL MATCH (category:DiscourseCategory)-[:HAS_TOPIC]->(topic)
+        # RETURN
+        #   id(post) AS post_id,
+        #   id(category) AS category_id,
+        # """
+        # We are matching topics by topic.HAS_POST post.id
+        # query = """
+        # UNWIND $post_ids AS post_id
+        # MATCH (post:DiscoursePost)
+        # WHERE id(post) = post_id
+        # MATCH (topic:DiscourseTopic)-[:HAS_POST]->(post)
+        # OPTIONAL MATCH (category:DiscourseCategory)-[:HAS_TOPIC]->(topic)
+        # RETURN
+        # id(post) AS post_id,
+        # id(category) AS category_id,
+        # """
+        # Amin suggested optimal one
+        # With neo4j automatically generated ids
+        # query = """
+        # MATCH (post:DiscoursePost)
+        # WHERE id(post) in $post_ids
+        # MATCH (topic:DiscourseTopic)-[:HAS_POST]->(post)
+        # OPTIONAL MATCH (category:DiscourseCategory)-[:HAS_TOPIC]->(topic)
+        # RETURN
+        # id(post) AS post_id,
+        # id(category) AS category_id
+        # """
+        # Without neo4j automatically generated ids
+        query = """
+        MATCH (post:DiscoursePost)
+        WHERE post.id in $post_ids
+        MATCH (topic:DiscourseTopic)-[:HAS_POST]->(post)
+        OPTIONAL MATCH (category:DiscourseCategory)-[:HAS_TOPIC]->(topic)
+        RETURN
+        post.id AS post_id,
+        category.id AS category_id
+        """
+        with self.driver.session() as session:
+            result = session.run(query, post_ids=post_ids)
+            records = [record.data() for record in result]
+            # print(f"Cathegories fetched: ", records)
+            self.driver.close()
+            # print(f"Number of records fetched in fetch_post_categories: {len(records)}")
+            return records
+    
+    def get_latest_post_created_at(self, forum_endpoint: str) -> Optional[str]:
+        """
+        Fetches the created_at timestamp of the latest post from a specified forum.
+
+        Args:
+            uri (str): The URI to connect to the Neo4j database.
+            user (str): The username for the Neo4j database.
+            password (str): The password for the Neo4j database.
+            forum_endpoint (str): The endpoint of the Discourse forum.
+
+        Returns:
+            Optional[str]: The created_at timestamp of the latest post, or None if no posts are found.
+        """
+        # With neo4j automatically generated ids
+        # query = """
+        # MATCH (forum:DiscourseForum {endpoint: $forum_endpoint})
+        # WITH forum
+        # MATCH (topic:DiscourseTopic {forumUuid: forum.uuid})
+        # MATCH (topic)-[:HAS_POST]->(post:DiscoursePost)
+        # RETURN post.createdAt AS created_at
+        # ORDER BY post.createdAt DESC
+        # LIMIT 1
+        # """
+        # Without neo4j automatically generated ids
+        query = """
```

Comment:
I don't see any difference of this query from previous commented one. Please remove the duplicate commented query.